spi/data-plane/data-plane-spi/src/main/java/org/eclipse/edc/connector/dataplane/spi/DataPlaneApiUrl.java

```diff
@@ -0,0 +1,27 @@
+/*
+ *  Copyright (c) 2020-2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.connector.dataplane.spi;
+
+
+import java.net.URL;
+
+@FunctionalInterface
+public interface DataPlaneApiUrl {
+
+    /**
+     * gets the URL which the HTTP Data Plane API provides.
+     */
+    URL get();
+}
```

Comment:
The Data Plane exposes two API:
 - a public API (the one you tackled in this PR)
 - a control API

Either we should have two methods in this interface (getPublic, getControl) or we should have two interfaces and then DataPlaneApiExtension should register them both

Reply:
I would rename this in `DataPlanePublicApiUrl` since for now only this  one is needed